src/main/java/ugh/fileformats/mets/MetsModsImportExport.java

```diff
@@ -71,12 +78,30 @@
 import ugh.dl.MetadataType;
 import ugh.dl.Person;
 import ugh.dl.Prefs;
-import ugh.exceptions.DocStructHasNoTypeException;
 import ugh.exceptions.ImportException;
-import ugh.exceptions.MetadataTypeNotAllowedException;
-import ugh.exceptions.PreferencesException;
-import ugh.exceptions.ReadException;
-import ugh.exceptions.WriteException;
+
+/*******************************************************************************
+ * ugh.fileformats.mets / MetsModsImportExport.java
+ *
+ * Copyright 2010 Center for Retrospective Digitization, Göttingen (GDZ)
+ *
+ * http://gdz.sub.uni-goettingen.de
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 3 of the License, or (at your
+ * option) any later version.
+ *
+ * This Library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this library. If not, see <http://www.gnu.org/licenses/>.
+ ******************************************************************************/
+
+import gov.loc.mods.v3.ModsDocument;
```

Comment:
Why did you move this import statement out of alphabetical ordering?

Reply:
Same as above.